app/assets/javascripts/miq_flash.js

```diff
@@ -35,6 +39,15 @@ function add_flash(msg, level, options) {
   var alertDiv = $('<div class="' + cls.alert + '"><button class="close" data-dismiss="alert"><span class="pficon pficon-close"></span></button></div>');
   alertDiv.append(iconSpan, textStrong);
 
+  if (options.long_alert) {
+    var detailsButton = $('<div style="padding-top: 4px"><button id="alert_expand_button" class="btn btn-default pull-right" type="submit">Details</button></div>');
```

Comment:
String should be extracted, also afaiu @serenamarie125 suggested to use the "see more" text vs "details"?